api/src/controllers/auth.js

```diff
@@ -577,3 +520,5 @@ function hashSha512ToHex(text) {
         .update(text)
         .digest('hex');
 }
+
+exports.createUploadPathIfNotExist = createUploadPathIfNotExist();
```

Comment:
I think you wanted to write `createUploadPathIfNotExist` instead of `createUploadPathIfNotExist()`.
In your code you call the function `createUploadPathIfNotExist` which returns a `Promise<string>` and export that `Promise<string>`.